nextflow_schema.json

```diff
@@ -9,22 +9,21 @@
             "title": "Input/output options",
             "type": "object",
             "fa_icon": "fas fa-terminal",
-            "description": "Define where the pipeline should find input data and save output data.",
+            "description": "Define where the pipeline should find input data and save output. data.",
```

Comment:
Minor typo here, `save output. data.` &rarr; `save output data.`

Reply:
> LGTM, found a couple of minor style issues

Thanks for you quick review!

I don't understand your suggestion here. In the orignial, I already have `save output data.`.

Reply:
I am confused, why your suggestions are in red. Or your references. Anyhow, you are right.